docs/macios/system-special-folders.md

```diff
@@ -0,0 +1,112 @@
+| `InternetCache`                      | $HOME/Library/Caches              | $HOME/Library/Caches          |
+| `LocalApplicationData`               | $HOME/Library/Application Support | $HOME/Documents               |
+| `LocalizedResources`                 |                                   |                               |
+| `MyComputer`                         |                                   |                               |
+| `MyDocuments`                        | $HOME/Documents                   | $HOME/Documents               |
+| `MyMusic`                            | $HOME/Music                       | $HOME/Documents/Music         |
+| `MyPictures`                         | $HOME/Pictures                    | $HOME/Documents/Pictures      |
+| `MyVideos`                           | $HOME/Movies                      | $HOME/Documents/Videos        |
+| `NetworkShortcuts`                   |                                   |                               |
+| `Personal`                           | $HOME/Documents                   | $HOME/Documents               |
+| `PrinterShortcuts`                   |                                   |                               |
+| `ProgramFiles`                       | /Applications                     | $HOME/Applications            |
+| `ProgramFilesX86`                    |                                   |                               |
+| `Programs`                           |                                   |                               |
+| `Recent`                             |                                   |                               |
+| `Resources`                          |                                   | $HOME/Library                 |
+| `SendTo`                             |                                   |                               |
+| `StartMenu`                          |                                   |                               |
+| `Startup`                            |                                   |                               |
+| `System`                             | /System                           |                               |
+| `SystemX86`                          |                                   |                               |
+| `Templates`                          | $HOME/Templates                   | $HOME/Documents/Templates     |
+| `UserProfile`                        | $HOME                             | $HOME                         |
+| `Windows`                            |                                   |                               |
+
+The returned system paths for `ApplicationData`, `Desktop`, `DesktopDirectory`, `Fonts`, `LocalApplicationData`, `MyMusic`, `MyPictures`, `MyVideos`, `ProgramFiles`, `System`, `Templates` differ beween macOS and Mac Catalyst. This is because by design Mac Catalyst matches the behavior of iOS.
```

Comment:
```suggestion
The returned system paths for `ApplicationData`, `Desktop`, `DesktopDirectory`, `Fonts`, `LocalApplicationData`, `MyMusic`, `MyPictures`, `MyVideos`, `ProgramFiles`, `System`, `Templates` differ between macOS and Mac Catalyst. This is because by design Mac Catalyst matches the behavior of iOS.
```

@davidbritch just a minor detail :)

Reply:
I'll fix this.

Reply:
Thanks a lot!